OpenRA.Game/WPos.cs

```diff
@@ -29,8 +29,8 @@ public struct WPos : IScriptBindable, ILuaAdditionBinding, ILuaSubtractionBindin
 
 		public static explicit operator WVec(WPos a) { return new WVec(a.X, a.Y, a.Z); }
 
-		public static WPos operator +(WPos a, WVec b) { return new WPos(a.X + b.X, a.Y + b.Y, a.Z + b.Z); }
-		public static WPos operator -(WPos a, WVec b) { return new WPos(a.X - b.X, a.Y - b.Y, a.Z - b.Z); }
+		public static WPos operator +(WPos a, in WVec b) { return new WPos(a.X + b.X, a.Y + b.Y, a.Z + b.Z); }
```

Comment:
I suspect a very large performance improvement. Unless optimization in the past already took care of this.

Reply:
Both can't be passed by reference?
Wouldn't it then be better to create a non static `operator +` also (if possible in c#).